modules/offers/client/components/OffersPresentational.js

```diff
@@ -0,0 +1,276 @@
+import React, { Component } from 'react';
+import { Button, DropdownButton, MenuItem, ButtonGroup, Tooltip } from 'react-bootstrap';
+import '@/config/client/i18n';
+import { limitTo, sanitizeHtml } from '../../../utils/filters';
+import OfferLocation from './OfferLocation.component';
+
+import PropTypes from 'prop-types';
+import { withTranslation } from 'react-i18next';
+
+export class OffersPresentational extends Component {
+  constructor(props) {
+    super(props);
+    this.renderOffer = this.renderOffer.bind(this);
+    this.renderButtonOwn = this.renderButtonOwn.bind(this);
+    this.renderHostingYesMaybe = this.renderHostingYesMaybe.bind(this);
+    this.setOfferDescriptionToggle = this.setOfferDescriptionToggle.bind(this);
+    this.state = {
+      offerDescriptionToggle: false,
+      isLoading: true,
+      isMobile: window.navigator.userAgent.toLowerCase().indexOf('mobile') >= 0 || window.isNativeMobileApp
+    };
+  }
+
+  /* Action Functions */
+  setOfferDescriptionToggle(toggleState) {
```

Comment:
Since we only ever toggle this to `true`, we can maybe simplify to:
```js
toggleLongerDescription() {
  this.setState({ showLongerDescription: true });
}
```